Automattic/jetpack
# VideoPress: set v1.1/videos endpoint as global when requesting video data

This PR fixes an issue (in dotcom) when requesting video data for the VideoPress video block.
For this, It defines the `v1.1/videos/<guid>` endpoint as global to work correctly.

Fixes https://github.com/Automattic/jetpack/issues/27796

#### Changes proposed in this Pull Request:

* VideoPress: set v1.1/videos endpoint as global when requesting video data

#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?



#### Testing instructions:






* Test in a Simple site
* Go to the block editor
* Create/edit a video block instance
* Open the sidebar
* Confirm after these changes, `title` and `description` are not disabled/

before | after
-----|-----
<img width="941" alt="Screen Shot 2022-12-07 at 16 21 44" src="https://user-images.githubusercontent.com/77539/206233810-bf2a6c5e-84aa-4743-90ed-dbbceefaddb6.png"> | <img width="944" alt="Screen Shot 2022-12-07 at 16 20 45" src="https://user-images.githubusercontent.com/77539/206233834-b495c9cf-81aa-48b6-8535-a43859925c91.png">
